### PR TITLE
Check if syntaxtree is supported

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide

**Comment by jmarolf** [comment]
test vsi please

**Comment by jmarolf** [comment]
test vsi please

File: src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                Dim list = Await mockCompletionService.Object.GetCompletionListAsync(
+                    document:=document,
+                    position:=0,
+                    triggerInfo:=CompletionTriggerInfo.CreateInvokeCompletionTriggerInfo(),
+                    options:=Nothing,
+                    providers:=Nothing,
+                    cancellationToken:=Nothing)
+                Assert.NotNull(list)
+                Assert.NotEmpty(list.Items)
+                Assert.True(list.Items.Length = 1, "Completion list contained more than one item")
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb]: I'm pretty sure xunit has Assert.Single() which simplifies this line and the one above it.

**Comment by jasonmalinowski** [comment]
:+1: if everything passes.

**Pull request merged**